pgbot/utils/utils.py

```diff
@@ -308,3 +308,11 @@ async def coro_check_channels_permissions(
             await asyncio.sleep(0)
 
     return booleans
+
+
+def safe_dict(dict_to_get, key, val_to_override):
```

Comment:
This is not needed, because one can always do `dict.get(key, [optional val if key is not found])`

Reply:
Ooh, I forgot about dict.get! I'll do that real quick